gradio/blocks.py

```diff
@@ -192,11 +192,18 @@ def __get__(self, instance, type_):
         return descr_get(instance, type_)
 
 
+@document()
 def update(**kwargs) -> dict:
     """
-    Updates component parameters
-    @param kwargs: Updating component parameters
-    @return: Updated component parameters
+    Updates component parameters.
```

Comment:
We should also explain that `gradio.update()` is shorthand for `gradio.[component].update()`, otherwise this is kind of confusing..

Reply:
please also add demos to the docstring. 

`Demos: blocks_essay`

(and any other demos we should embed in the Try Examples section.)

Reply:
Great feedback @aliabd ! 

I added the demos to the docstring and made some changes to the documentation generator so that kwargs are properly displayed if they are documented:

![image](https://user-images.githubusercontent.com/41651716/178346977-fd54e270-7adf-4d3d-b79f-6497676e7090.png)